scripts/utils.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+arg2nz() { test $# -ge 2 -a ! -z "$2" || usage "empty value for" "$1"; }
```

Comment:
@marcin-mazurek could you please add a comment or add func name because its not clear  what's this script is doing (plus i'm not expert on bash script 😅)

Reply:
This is transferring all your coins to him 😈

Reply:
Good catch, although I just moved this function I've renamed it to something that makes more sense :) It essentially validates arguments in the following format `--build-id 20131` and ensures the second part is provided or calls the `usage` function that shows the script help.